themes/startwords/theme.toml

```diff
@@ -1,12 +1,12 @@
-name = "Blank"
+name = "startwords"
 license = "MIT"
-licenselink = "https://github.com/vimux/blank/blob/master/LICENSE"
-description = "Starter Hugo theme for developers."
+licenselink = "https://github.com/Princeton-CDH/startwords/blob/master/themes/startwords/LICENSE"
+description = "Theme for the Startwords research periodical."
 homepage = "https://github.com/vimux/blank/"
-tags = ["blog", "plain", "blank", "starter", "development"]
+tags = ["journal", "research", "periodical"]
 features = ["blog"]
 min_version = "0.20"
 
 [author]
-  name = "Vimux"
-  homepage = "https://github.com/vimux"
+  name = "Center for Digital Humanities @ Princeton"
+  homepage = "https://github.com/Princeton-CDH"
```

Comment:
I wonder if author homepage should be cdh.princeton.edu instead?

Reply:
it definitely should, good idea!